src/native/libs/build-local.sh

```diff
@@ -4,24 +4,25 @@
 # The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Executable bit was dropped during the move: `chmod +x src/native/libs/build-local.sh`

Reply:
Thank you.